projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -225,9 +206,28 @@ export class ExplorerComponent {
     this.preferenceService.set(ExplorerComponent.RESULTS_EXPANDED_PREFERENCE, expanded);
   }
 
+  protected buildContextItems(): ExplorerContextToggleItem[] {
```

Comment:
Why build them instead of set them directly like before?

Reply:
Yep. Consumers could extend it anyway they want :)